engine/gamesys/src/gamesys/scripts/script_resource.cpp

```diff
@@ -1783,6 +1826,21 @@ static int GetRenderTargetInfo(lua_State* L)
             lua_pushinteger(L, color_buffer_flags[i]);
             lua_setfield(L, -2, "buffer_type");
```

Comment:
nitpick: for readability, perhaps use "lua_pushinteger(L, buffer_type);" for readability/consistency